content/en/software/using-medley/_index.md

```diff
@@ -93,7 +75,39 @@ These documents were converted from Medley's internal format into PDFs. Watch ou
 ### Interlisp Books
 
 - [Interlisp - The Language and Its Usage](/documentation/1986-interlisp-language-book-1.pdf)
-   Now also available in [zipped DJVU format](/documentation/1986-Interlisp-Language-Usage.ocr.djvu.zip)
+  Also available in [zipped DJVU format](/documentation/1986-Interlisp-Language-Usage.ocr.djvu.zip)
   - <a href="20211225-interlisp-book-2.pdf">Medley Interlisp: Interactive Programming Environment (derived from Interlisp-D)</a>
   - <a href="2021-interlisp-book-3.pdf">Medley Interlisp: Interactive Programming Tools (derived from Interlisp-D)</a>
 - <a href="IRM.pdf">(1993) Interlisp Reference Manual</a>
+
+# Another brief guideline for online users
+
+## Starting up
+
+The Medley Desktop at startup contains 4 windows of interest:
+
+* Prompt Window: The black window at the top of the screen. It is used to display system or application prompts
+* Exec (INTERLISP or XCL) window: The main window where you run functions and develop programs.
+* Medley logo window: A window containing the Interlisp Medley logo as a bit map.
+* Status Bar window
+
+Medley Online users get some additional buttons on the right-hand side of the screen.
+
+### Managing memory images and sessions
+
+Medley has a byte-coded virtual machine architecture with a VM designed for Lisp.  This is a common practice, used by Javascript, Flash and Java and other systems to aid in portability: to support a new architecture, you just port the "virtual machine emulator" rather than the whole system.
+
+With Medley, there are two types of files relevant to managing memory images and updating them across sessions: `lisp.virtualmem` and `.sysout`.
+
+The `lisp.virtualmem` file is a capture of the "current" state of the system (i.e., it is a copy of the virtual memory at a point in time). `lisp.virtualmem` is written whenever you execute `(IL:LOGOUT)` and also whenever executing `(IL:SAVEVM)`. You can restart Medley using a `lisp.virtualmem` and it will pick up essentially where it left off before the `LOGOUT` or `SAVEVM` (with the exception that the user can set `BEFORE`/`AFTER` and `LOGOUT`/`SAVEVM` code that runs before you get control of the restarted `lisp.virtualmem`).
+
+A `.sysout` is a virtual memory image produced by `MAKESYS` (for writing an image for distribution) and `SYSOUT` (for saving a named checkpoint, e.g. to revert to a previous state if needed), which differ only in the way they process the startup options. You can (and most frequently do) start Medley from a sysout file. When Medley first starts from a sysout, it automatically runs initialization scripts — a site initialization script followed by a per-user initialization script (if available).
```

Comment:
How do the sysouts produced by `MAKESYS` and `SYSOUT` differ in the way the process the startup options?

Reply:
MAN MAKESYS vs MAN SYSOUT tells the original story.
In Medley, we've tried to keep the sense of the difference between MAKESYS, SYSOUT, and SAVEVM with respect to handling of open files, user GREETing.

Reply:
does this need more explanation? or rewrite to not use the function names such as MAKESYS and explain the concept ?

Reply:
To me the explanation is clear as is. It's useful to keep the function names so that interested users can look them up.